Autotests/Lesson1_1/Program.cs

```diff
@@ -4,18 +4,36 @@ namespace AutotestLesson1
 {
     class Program
     {
+        
+
         static void Main(string[] args)
         {
+            bool isInt(string value)
```

Comment:
@MariaOsminkina Very strange implementation, you already have 
`out int res`
Why not use the result ?
Please refactor this method and it's usings